packages/core/css/type.module.css

```diff
@@ -13,6 +13,74 @@
   --psTypeFontSizeSmall: 14px;
   --psTypeFontSizeXSmall: 12px;
```

Comment:
Do you think the old font sizes, weights and line heights should be removed?  If not, why keep?